flocker/control/__init__.py

```diff
@@ -0,0 +1,5 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+Control service for managing a Flocker cluster.
```

Comment:
This probably merits a more expansive description, or at least a link to some other architecture/design documentation (but I don't think we have any linkable right now).